faqs/stocks/which-screener.md

```diff
@@ -9,75 +9,122 @@ description: >-
+The PE based on last 5 years average earnings will be 8.29. This is what Benjamin Graham recommends to use in such cases.
 
-![ROIC of Abbott India on Tijori Finance - Dark Mode](../../.gitbook/assets/tijori-roic-dark.png)
+----
 
-![ROIC of Abbott India on Tijori Finance - Light Mode](../../.gitbook/assets/tijori-roic-light.png)
+## Operating margins and gross margins
 
-### Some Causes of these inaccuracies
+The text-book definition of gross profit margin (GPM) is `sales - cogs`.
 
-One of the components in the calculation of ROIC is Capital Invested and it excludes all non-operational assets that a company has. Cash and Cash Equivalents are generally not considered operational assets, but Screener.in and Morning Star don’t seem to consider this factor for some reason. Since they end up including Cash and Cash Equivalents in Capital Invested, ROIC is incorrectly penalized for companies with high amounts of cash on hand.
+`COGS` is cost of goods sold.
 
-ROIC is also a bit hard to calculate and get right so this may have been another factor for inaccuracy. Some screeners don’t even include ROIC for this reason.
+Most websites will show the gross margins on this standard definition. However, this can sometimes be misleading.
+
+Example in case of TCS - `screener.in` shows the GPM as 100%. This is because TCS has no inventory. But the correct metric for `COGS` in this case should be their employee cost.
+
+Similarly in case of banks, most websites (eg `screener.in`), don't consider their interest cost in OPM (operating profit margin) and GPM (gross profit margin). Thus these numbers are shown exceptionally high.
+
+Screening companies on margins can often yield incorrect results for:
+- Insurance companies
+- Banks
+- Finance companies
+- Mining companies
+
+----
+
+## Return ratios
+
+In case of Abbott India, the ROIC reported as on 29th April 2021 is:
+
+```
+Screener.in:        22.90%
+MorningStar.in:     21.53%
+TijoriFinance.com:  131%
+Correct value:     ~138%
+```
+
+The company held fixed-deposits (cash equivalents) of ₹ 2,197 Cr in FY20. While `TijoriFinance` excluded these cash-equivalents in the calculation of capital employed, other two websites didn't exclude it.
+
+These differences in the methods of calculating ROCE, ROIC, ROA and other return ratios can yield wildly different results. You can sometimes miss some interesting companies because their calculated return might be much lower than their economic return ratios.
+
+----
 
 ## Summary and Wrap-up
 
-Every screener is bound to get inaccuracies whether it's screener.in, MorningStar, Tijori Finance, or some new entrant in this space.
+The auto-calculated ratios on websites can often be different from manual calculations. Those calculations will often miss sector specific adjustments or "special cases" which are too frequent in finance world. Screening and relying only on auto-calculated ratios can thus be *injurious*.
```

Comment:
We should add the summary we had discussed yesterday, that it's not possible to easily write software that automates the computations of different metrics, across thousands of listed companies, considering all nuances.

It's up to the investor using the screener to cross-check the computations, and understand how the software arrived at that number. This should be part of due-diligence for an investor.

Reply:
Have added a paragraph for this in https://github.com/indiainvestments/content/pull/49/commits/ff0ec4fd12010aff5dafc1e6cfebea4eef9d4517

Didn't add too much details about all the difficulties because they might just be a rant for us. It might not add too much value for a investor.